paper-menu-shared-styles.html

```diff
@@ -41,6 +41,11 @@
       .selectable-content > ::content > *[colored]:focus:after {
         opacity: 0.26;
       }
+
+      .selectable-content > ::content > * > div,

```

Comment:
@rubenstolk should add `selectable-content > ::content > * > iron-icon,` to handle this scenario used in PSK.

```
<paper-menu class="list" attr-for-selected="data-route" selected="[[route]]">
  <a data-route="home" href="/" on-tap="onDataRouteClick">
      <iron-icon icon="home"></iron-icon>
      <span>Home</span>
  </a>
...
```
This will handle when tapping on icon.